### PR TITLE
Customize color and icon on the querybar

### PR DESCRIPTION
**Pull request by francisl**
Add more option for configuration

Enable KidsFirst design
Consider cqdg design https://www.figma.com/file/cnK68RMYFGoLbgsnCYvoo6/FerlabUI?node-id=647%3A10344

Screen Shot 2021-03-30 at 11 36 43 AM

### CODE REVIEW

File: packages/style/components/queryBuilder/QueryBar.module.scss [hunk 1]
```diff
@@ -65,6 +67,15 @@
             >  :first-child {
                 margin-right: .5rem;
             }
+
+            img,svg {
+                height: 22px;
```
Comment (kjacqueschu) [inline packages/style/components/queryBuilder/QueryBar.module.scss:72]: you cant hardcod height of the icons in CQDG it is 18px. And not sure why we put this css here, you have full control of the icon pass down to Querybuilder, you can add a classname and style it outside of the queryBuilder

File: packages/style/components/queryBuilder/QueryPill.module.scss [hunk 2]
```diff
@@ -20,7 +21,7 @@
     }
 
     .close {
-        padding: .3rem .2rem .3rem .4rem;
+        padding: 0 .2rem 0 .4rem;
```
Comment (kjacqueschu) [inline packages/style/components/queryBuilder/QueryPill.module.scss:24]: I styled the x close button with Lucas and this was what he wanted with th .3rem .2rem .3rem .4rem. why are we removing the .3rem ?

**Review by kjacqueschu — CHANGES_REQUESTED** [review]

File: packages/style/components/queryBuilder/QueryPill.module.scss (hunk 2 above)
Comment (francisl) [reply packages/style/components/queryBuilder/QueryPill.module.scss:24]: if you look at the design, it breaks his own line-height

File: packages/style/components/queryBuilder/QueryBar.module.scss (hunk 1 above)
Comment (francisl) [reply packages/style/components/queryBuilder/QueryBar.module.scss:72]: my bad here, fixing it

**Review by kjacqueschu — APPROVED** [review]

**Pull request merged**